api/room_resource/schema.py

```diff
@@ -290,6 +297,25 @@ def resolve_get_resources_by_room_id(self, info, room_id):
             resources.append(room_resource)
         return RoomResources(roomResources=resources)
 
+    @Auth.user_roles('Admin')
+    def resolve_resource_by_name(self, info, name):
```

Comment:
Good work @mnswaleh, works as expected, just remove the unnecessary blank lines in this function and you should be good to go, line:`306,313,316`